_data/toc.yaml

```diff
@@ -1646,6 +1646,30 @@ manuals:
       title: Compatibility
     - path: /registry/help/
       title: Getting help
+
+- sectiontitle: Atomist
```

Comment:
hmmm, I don't think that Docker Hub is the right place. Atomist integrates with Hub, but it is a separate platform. Though I am not happy with the current placement either... It might fit under the new 'trusted content' section, but until then I think it's better to maybe keep it separate? And it is probably fine if it gets a bit lost currently, since it is still early access days..

Reply:
Sorry, i didn't make that very clear. I meant beneath the Docker Hub section, not within the Docker Hub section

Reply:
Aha, got it! I can do that :)